media-libs/pnglite/pnglite-0.1.17.recipe

```diff
@@ -54,6 +54,7 @@ INSTALL()
 	mkdir -p $includeDir $libDir
 	cp pnglite.h $includeDir/
 	cp libpnglite.so.0 $libDir/
+	ln -sr $libDir/libpnglite.so.0 $libDir/libpnglite.so
```

Comment:
Shouldn't this be: `ln -sr $libDir/libpnglite.so.0 $developLibDir/libpnglite.so` ?

Reply:
Strike that, looks OK :)